### PR TITLE
 Ticket 7 POST comment completed with testing

### PR DESCRIPTION
**Pull request by J-PWalk**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -158,4 +158,60 @@ describe("/api/reviews/:review_id/comments", () => {
     .then(({body})=>{
       expect(body.comments).toEqual([])
     })
+    
 });
+
+describe('METHOD: POST', () => {
```
Comment (aloush-dev) [inline __tests__/app.test.js:164]: I'd add two more tests, one that ignore any unnecessary properties on an objects.
and one if the expected properties are missing from the object.

File: db/Controllers/comments.controller.js [hunk 2]
```diff
@@ -15,12 +18,12 @@ exports.getComments = (req, res, next) => {
...
-exports.postReviewComment = (req, res, next) => {
-  const {reviewID} = req.params
-  const {body} = req
-
-  fetchPostComment(reviewID, body).then(([comment])=>{
+exports.postComment = (req, res, next) => {
+  const { review_id } = req.params;
+  const { body } = req
+console.log("in controller")
+console.log(body)
```
Comment (aloush-dev) [inline db/Controllers/comments.controller.js]: remember to remove console logs

File: db/Models/comments.model.js [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
   });
-}
\ No newline at end of file
+};
+
+exports.fetchComment = (review_id, commentObj) => {
+  const params = [commentObj.username, commentObj.body, review_id]
+  let sqlQuery = `INSERT INTO comments (author, body, review_id) VALUES ($1,$2,$3) RETURNING *;`
+  console.log("in model")
+  // console.log(review_id)
```
Comment (aloush-dev) [inline db/Models/comments.model.js]: and remove commented out code

**Review by aloush-dev — CHANGES_REQUESTED** [review]
Nice job here please include the 2 tests that I have mentioned

**Comment by Harrison365** [comment]
Great work. Just need a 404 test for a valid article_id that doesn't exist.

**Pull request merged**